server/controllers/incidents.js

```diff
@@ -1,5 +1,6 @@
 const Incident = require("../models").Incidents;
 const User = require("../models").Users;
+const Note = require("../models").Notes;
```

Comment:
Why are we using the singular form instead of the plural form? `Notes`

Reply:
The methods in each controller define what services a model uses. This is why we define the constant in singular form to reference the model. For example; the method `create` is meant to serve as a POST for a Note vs Notes. This is the convention we are using.

Reply:
👍